crates/superposition_types/src/database/models/experimentation.rs

```diff
@@ -12,7 +12,7 @@ use superposition_derives::{JsonFromSql, JsonToSql};
 use crate::{Condition, Exp, Overrides};
 
 #[cfg(feature = "diesel_derives")]
-use super::schema::*;
+use super::super::schema::*;
```

Comment:
is this correct ?
`super::super` seems weird

Reply:
Yeah it is, that is why the tests passed. 
I thought it was weird too, but since it's inside models  it needs to go one level up to access